opencti-platform/opencti-graphql/package.json

```diff
@@ -20,7 +20,7 @@
     "start:cluster": "NODE_ENV=cluster node build/back.js",
     "serv": "node build/back.js",
     "insert:dev": "node build/script-insert-dataset.js --datasets=DATA-TEST-STIX2_v2,poisonivy",
-    "insert:dev:e2e": "ELASTICSEARCH__INDEX_PREFIX=e2e node build/script-insert-dataset.js --datasets=DATA-TEST-STIX2_v2,data-test-stix-e2e",
+    "insert:dev:e2e": "ELASTICSEARCH__INDEX_PREFIX=e2e node build/script-insert-dataset.js --datasets=DATA-TEST-STIX2_v2,data-test-stix-e2e,poisonivy",
```

Comment:
Is the script used elsewhere? Would it be better to write the dataset directly into the script?

Reply:
It is the same script used for insert data for testing purpose and for local environment (cf insert:dev) that's why now we have an argument to tell which dataset to use